docs/excel/custom-functions-errors.md

```diff
@@ -51,19 +50,19 @@ let error = new CustomFunctions.Error(CustomFunctions.ErrorCode.invalidNumber);
 throw error;
 ```
 
-When you return a `#VALUE!` error you can also include a custom message that will be shown in a popup when the user hovers over the cell. The following example shows how to return a custom error message.
+The `#VALUE!` and `#N/A` errors also support custom error messages. Custom error messages will be displayed in the error indicator menu, which is accessed by hovering over the error flag on each cell with an error. The following example shows how to return a custom error message with the `#VALUE!` error.
```

Comment:
I tend to favor present tense over future. YMMV, but I think it's more concise and clean.
```suggestion
The `#VALUE!` and `#N/A` errors also support custom error messages. Custom error messages are displayed in the error indicator menu, which is accessed by hovering over the error flag on each cell with an error. The following example shows how to return a custom error message with the `#VALUE!` error.
```